rdtools/analysis.py

```diff
@@ -455,23 +470,31 @@ def srr_soiling(self, aggregated, aggregated_insolation, **kwargs):
 
     def sensor_preprocess(self):
         '''
-        Perform sensor-based normalization, filtering, and aggregation work flow.
+        Perform sensor-based normalization, filtering, and aggregation.
+        If self.power_expected is set manually prior to this process, 
+        normalize_with_expected_power will be used instead of pvwatts.
         '''
         if self.poa is None:
             raise ValueError('poa must be available to perform sensor_preprocess')
         if self.cell_temperature is None and self.ambient_temperature is None:
             raise ValueError('either cell or ambient temperature must be available to perform sensor_preprocess')
         if self.cell_temperature is None:
             self.cell_temperature = self.calc_cell_temperature(self.poa, self.windspeed, self.ambient_temperature)
```

Comment:
Are the thermal model inputs needed for anything if the user provides expected power?  If not then maybe these should be moved into the `if self.power_expected is None` block.